config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml

```diff
@@ -0,0 +1,216 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack-network-isolation
+spec:
+  secret: osp-secret
+  storageClass: local-storage
+  cinder:
+    template:
+      cinderAPI:
+        containerImage: quay.io/tripleozedcentos9/openstack-cinder-api:current-tripleo
+        externalEndpoints:
+        - endpoint: internal
+          ipAddressPool: internalapi
+          loadBalancerIPs:
+          - 172.17.0.80
+      cinderBackup:
+        containerImage: quay.io/tripleozedcentos9/openstack-cinder-backup:current-tripleo
+        customServiceConfig: |
+          [DEFAULT]
+          backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
+          backup_ceph_pool = backups
+          backup_ceph_user = openstack
+        networkAttachments:
+        - storage
+      cinderScheduler:
+        containerImage: quay.io/tripleozedcentos9/openstack-cinder-scheduler:current-tripleo
+      cinderVolumes:
+        volume1:
+          containerImage: quay.io/tripleozedcentos9/openstack-cinder-volume:current-tripleo
+          customServiceConfig: |
+            [DEFAULT]
+            enabled_backends=ceph
+            [ceph]
+            volume_backend_name=ceph
+            volume_driver=cinder.volume.drivers.rbd.RBDDriver
+            rbd_ceph_conf=/etc/ceph/ceph.conf
+            rbd_user=openstack
+            rbd_pool=volumes
+            rbd_flatten_volume_from_snapshot=False
+            rbd_secret_uuid=_FSID_
```

Comment:
Eventually, you should be able to remove line 41 `rbd_secret_uuid=_FSID_` because of this patch:

  https://review.opendev.org/c/openstack/cinder/+/860090

@Akrog @ASBishop We need to wait for Antelope before we can do this, right?

Oddly, I left it out and cinder worked for me with zed.

  https://github.com/fultonj/zed/blob/main/edpm/kustomize/kustomization.yaml#L39

I don't understand why yet however, so I think we should keep this as @stuggi has it. I was using zed

    image: quay.io/tripleozedcentos9/openstack-cinder-volume:current-tripleo

Reply:
@fultonj Not sure of what you were testing when you wrote "cinder worked for me with zed," but bear in mind the rbd_secret_uuid is required for nova's side of things. Without that option, you can create volumes but nova won't be able to attach them.

At present, that patch is only available in Antelope. I'll ask around and see how cinder people feel about a stable backport.